src/PruneUnusedImagesModal.jsx

```diff
@@ -49,81 +49,71 @@ function ImageOptions({ images, checked, isSystem, handleChange, name, showCheck
+    const systemImages = unusedImages.filter(image => image.isSystem);
+    const showCheckboxes = userImages.length > 0 && systemImages.length > 0;
 
-    render() {
-        const isSystem = this.props.systemServiceAvailable;
-        const userImages = this.props.unusedImages.filter(image => !image.isSystem);
-        const systemImages = this.props.unusedImages.filter(image => image.isSystem);
-        const showCheckboxes = userImages.length > 0 && systemImages.length > 0;
-        return (
-            <Modal isOpen
-                   onClose={this.props.close}
-                   position="top" variant="medium"
-                   title={cockpit.format(_("Prune unused images"))}
-                   footer={<>
-                       <Button id="btn-img-delete" variant="danger"
-                               spinnerAriaValueText={this.state.isPruning ? _("Pruning images") : undefined}
-                               isLoading={this.state.isPruning}
-                               isDisabled={!this.state.deleteUserImages && !this.state.deleteSystemImages}
-                               onClick={this.handlePruneUnusedImages}>
-                           {this.state.isPruning ? _("Pruning images") : _("Prune")}
-                       </Button>
-                       <Button variant="link" onClick={() => this.props.close()}>{_("Cancel")}</Button>
-                   </>}
-            >
-                <Flex flex={{ default: 'column' }}>
-                    {isSystem && <ImageOptions
-                  images={systemImages}
-                  name="deleteSystemImages"
-                  checked={this.state.deleteSystemImages}
-                  handleChange={this.handleChange}
-                  showCheckbox={showCheckboxes}
-                  isSystem
-                    />
-                    }
-                    <ImageOptions
-                  images={userImages}
-                  name="deleteUserImages"
-                  checked={this.state.deleteUserImages}
-                  handleChange={this.handleChange}
-                  showCheckbox={showCheckboxes}
-                  isSystem={false}
-                    />
-                </Flex>
-            </Modal>
-        );
-    }
-}
+    return (
+        <Modal isOpen
+               onClose={close}
+               position="top" variant="medium"
+               title={cockpit.format(_("Prune unused images"))}
+               footer={<>
+                   <Button id="btn-img-delete" variant="danger"
+                           spinnerAriaValueText={isPruning ? _("Pruning images") : undefined}
+                           isLoading={isPruning}
+                           isDisabled={!deleteUserImages && !deleteSystemImages}
+                           onClick={handlePruneUnusedImages}>
+                       {isPruning ? _("Pruning images") : _("Prune")}
+                   </Button>
+                   <Button variant="link" onClick={() => close()}>{_("Cancel")}</Button>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1146-20221124-125453-c8adda44-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)